ui/v2.5/src/docs/en/Manual/Interface.md

```diff
@@ -26,6 +26,10 @@ The stash UI can be customised using custom CSS. See [here](https://github.com/s
 
 [Stash Plex Theme](https://github.com/stashapp/stash/wiki/Theme-Plex) is a community created theme inspired by the popular Plex interface.
 
+## Custom Javascript
+
+The stash support the inject of custom javascript. Be aware thet bad Javascript could break the UI.
```

Comment:
There's a few grammatical and spelling errors here.

Reply:
Thanks, I breezed through this update without looking too closely at the text. I've pushed an update for this.